src/models/NFTAttributes.ts

```diff
@@ -1,6 +1,43 @@
 import { RoyaltyAttributes } from '../nevermined'
 import { AssetAttributes } from './AssetAttributes'
 import { ERCType, NeverminedNFTType, NeverminedNFT1155Type, NeverminedNFT721Type } from './'
+import { ServiceType } from '../ddo/types'
+
+export class NFTServiceAttributes {
+  /**
+   * The asset is transferred (true) or minted (false) with Nevermined contracts
+   */
+  nftTransfer?: boolean
+
+  /**
+   * If true means the NFT works as a subscription
+   */
+  isSubscription?: boolean
+
+  /**
+   * If is a subscription this means the number of blocks the subscription last. If 0 means unlimited
+   */
+  duration?: number
+
+  /**
+   * Number of editions
+   */
+  amount?: bigint
+
+  static defaultValues = {
+    serviceType: 'nft-access' as ServiceType,
+    nftTransfer: true, // The NFT will use transfers
+    isSubscription: false, // By default the asset doesn't represent a subscription
+    duration: 0, // Because it's not a subscription it doesn't have a duration
```

Comment:
doesn't duration zero mean that it has unlimited duration?

Reply:
in in combination with `isSubscription = true`